config/sample.yml

```diff
@@ -42,6 +42,16 @@ messageStore: # Postgresql database configuration for message store
   password:
   url:
 
+contactDiscovery: # Configuration for interfacing with Contact Discovery Service cluster
```

Comment:
I think we should be referring to this service as the "directory," so maybe we should move this to the directory configuration object?

Reply:
Agreed. Working on it.

Reply:
done in e6308599137e103cb801635962c5af58a224c284